examples/tutorials/04_cubes_and_objects/09_custom_objects.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017 Anki, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the file LICENSE.txt or at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''This example demonstrates how you can define custom objects.
+
+The example defines several custom objects (2 cubes, a wall and a box). When
+Cozmo sees the markers for those objects he will report that he observed an
+object of that size and shape there.
+
+You can adjust the markers, marker sizes, and object sizes to fit whatever
+object you have and the exact size of the markers that you print out.
+'''
+
+import time
+
+import cozmo
+from cozmo.objects import CustomObject, CustomObjectMarkers, CustomObjectTypes
+
+
+def handle_object_appeared(evt, **kw):
+    # This will be called whenever an EvtObjectAppeared is dispatched - so
```

Comment:
I would remove the 'so' here can just leave it as "whenever an Object comes into view."